app/lib/utils.js

```diff
@@ -7,3 +9,7 @@ export const formatAttachmentUrl = (attachmentUrl, userId, token, server) => {
 	}
 	return encodeURI(`${ server }${ attachmentUrl }?rc_uid=${ userId }&rc_token=${ token }`);
 };
+
+export const lt = (currentServerVersion, oldServerVersion) => lowerThan(coerce(currentServerVersion), oldServerVersion);
```

Comment:
Currently, we're using the functions from `semver` directly, but that's not meaninful.
Can you rename `lt` to `isServerVersionLowerThan`?
Same for `gte`.

@gerzonc

Reply:
Alright